Week3/homework/src/createToDoList.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+function createToDoList(connection, errorHandler, req, res) {
+  const username = req.params.username;
+  const password = req.params.password;
```

Comment:
**Non-blocking:**

In real life projects, you should never store passwords as a plaintext: https://security.stackexchange.com/questions/120540/why-shouldnt-i-store-passwords-in-plaintext

Remember SQL Injection attack? if you store your passwords as plaintext, once they are stolen - they can be used for other services, because ppl tend to use the same password in multiple places.

Reply:
I searched for this Igor but i couldn't find any data type for passwords. I came across with sha like syntax but I am not sure if it is proper one ?